### DISCUSSION TITLE
Cannot assign to read only property '_status' of object '#<Object>'

### DISCUSSION THREAD

**Discussion by ECBSJ** [WalletConnect/walletconnect-monorepo#3620] [Developer Support]
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/61910547/e1554577-6b8f-4abe-805a-8a93971300c9)

I've migrated to v2 but am getting this error in my production build....

Has anyone encountered this issue as well?

**Comment by glitch-txs** [comment]
never seen it 🤔 could you share a [MRE](https://github.com/orgs/WalletConnect/discussions/3457)?
